HTI.md

```diff
@@ -264,6 +263,7 @@ The code fragment below shows the FHIR task (in gray) as part of the JWT message
       "reference": "Patient/9"
     }
   },
+  "fhir-v": "STU3",
```

Comment:
Is er een reden dit `fhir-v` te noemen en niet "gewoon" `fhir-version`?

Reply:
Ik ben zelf ook voorstander van `fhir-version`, maar ik heb me geprobeerd aan de [RFC-7519](https://tools.ietf.org/html/rfc7519#section-4.1) te houden:

>  Applications using JWTs should define which
   specific claims they use and when they are required or optional.  All
   the names are short because a core goal of JWTs is for the
   representation to be compact.

Zal ik de claim gewoon maar aanpassen naar `fhir-version`? Gok dat de impact op de HTI performance wel meevalt!

Reply:
Zouden 6 extra karakters een impact hebben? Ik denk van niet, maar ik snap dat je naar de RFC gekeken hebt - dat had ik dus niet gedaan ;)

Als je het //mij// vraagt, zou ik `fhir-version` gebruiken, maar het is totaal geen halszaak.

Reply:
Ik heb 'm aangepast!